enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,259 @@
+Is 5 minutes for important a target deployment image satisfactory for an IPI deployment. Additonally, is it even
+possible for the Power VS team to reduce the current import time from 30 minutes to 5 (for the purposes of meeting this
+requirement)?
+
+#### Deployment Time
+What is the maximum allowable time for a successful deployment to take. Is under an hour sufficient?
+
+#### DHCP in the IBM Cloud
+How soon will we have DHCP in the Power VS platform. If we can't get DHCP, do we have other options?
+
+#### Flavors or Granular Control
+One thing that is being discussed is whether it would be better to expose a subset of cloud instance "flavors" with
+pre-defined compute, storage, CPU cores, etc., or to allow the installer to expose granular control over each
+configurable item.
+
+### Test Plan
+
+Multi-Arch currently tests results by running upgrade and e2e tests with the OpenShift Prow CI environment. Our test
+plan would extend this to include these IPI scenarios.
+
+### Graduation Criteria
+
+- Our initial target is to work towards `Dev Preview` in OpenShift 4.9.
+- From `Dev Preview`, we will determine the target for `Tech Preview`.
+- From `Tech Preview`, we will determine the target for `GA`.
+- `Dev Preview`, `Tech Preview`, `GA` in OpenShift
+- [Deprecation policy][deprecation-policy]
+
+[deprecation-policy]: https://kubernetes.io/docs/reference/using-api/deprecation-policy/
+
+#### Dev Preview -> Tech Preview
+TODO - define graduation in detail
+
+- Ability to utilize the enhancement end to end
+- End user documentation, relative API stability
+- Sufficient test coverage
+- Gather feedback from users rather than just developers
+- Enumerate service level indicators (SLIs), expose SLIs as metrics
+- Write symptoms-based alerts for the component(s)
+
+#### Tech Preview -> GA
+TODO - define graduation in detail
+
+- More testing (upgrade, downgrade, scale)
+- Sufficient time for feedback
+- Available by default
+- Backhaul SLI telemetry
+- Document SLOs for the component
+- Conduct load testing
+
+**For non-optional features moving to GA, the graduation criteria must include
+end to end tests.**
+
+#### Removing a deprecated feature
+
+- Announce deprecation and support policy of the existing feature
+- Deprecate the feature
+
+### Upgrade / Downgrade Strategy
+
+All upgrade and downgrade expectations are the same for existing IPI deployments.
+
+### Version Skew Strategy
+
+This should be the same as other IPI deployments.
+
+## Implementation History
+
+Major milestones in the life cycle of a proposal should be tracked in `Implementation
+History`.
+
+## Drawbacks
+
+The greatest argument against IPI on Power VS would be opportunity cost. That said, for the Multi-Arch team, we believe
+this to be the best return on investment when compared to the many other platforms asked of our team.
+
+## Alternatives
+
+The are other options for supporting IPI for for power hardware. One obvious option is IPI on KVM via libvirt; however,
```

Comment:
Should we replace KVM via libvirt, with PowerVM/PowerVC?

Reply:
Made a note about PowerVM + VC as well.